### PR TITLE
kubeadm: update the docs for HA with --control-plane-endpoint

### PR DESCRIPTION
**Pull request by neolit123**
The addition of the flag "---control-plane-endpoint" for
"kubeadm init" removes the requirement for creating a config file
for the stacked etcd setup.

/hold
/assign @fabriziopandini @jimangel 
/sig cluster-lifecycle
/priority important-longterm
/kind feature

### CODE REVIEW

File: content/en/docs/setup/production-environment/tools/kubeadm/high-availability.md [hunk 1]
```diff
@@ -100,31 +100,17 @@ option. Your cluster requirements may need a different configuration.
...
-some like Weave do not. See the [CNI network documentation](/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/#pod-network).
-To add a pod CIDR set the `podSubnet: 192.168.0.0/16` field under
-the `networking` object of `ClusterConfiguration`.
-{{< /note >}}
-
 1.  Initialize the control plane:
 
     ```sh
-    sudo kubeadm init --config=kubeadm-config.yaml --upload-certs
+    sudo kubeadm init --kubernetes-version stable --control-plane-endpoint "LOAD_BALANCER_DNS:LOAD_BALANCER_PORT" --upload-certs
```
Comment (fabriziopandini) [inline content/en/docs/setup/production-environment/tools/kubeadm/high-availability.md]: If I'm not wrong, we can remove `--kubernetes-version` stable (it is the default)

**Comment by neolit123** [comment]
@fabriziopandini updated.

**Comment by fabriziopandini** [comment]
@neolit123 awesome!
/lgtm

**Pull request merged**